app/assets/stylesheets/custom.scss

```diff
@@ -291,3 +288,31 @@ ul, menu, dir {
 .filter {
   text-align: center;
 }
+
+// /* =========================
+//
+//      USER LISTINGS INDEX PAGE
+//
+//    =========================*/
+
+#img_container {
+  position:relative;
```

Comment:
Properties should be ordered display, position, text-align<br>Colon after property should be followed by one space